### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by atwoosnam**
Update install/uninstall directions in README to match wiki home page: https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/wiki

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,10 +15,23 @@ Tanzu Toolkit for Visual Studio is an extension for Visual Studio (2022 & 2019)
...
-- Download the `Tanzu.Toolkit.VisualStudio.vsix` file & double-click to prompt an installation into your Visual Studio instance.
-- To uninstall this extension from Visual Studio, visit the Visual Studio Extension Manager found under the `Extensions` menu.
+To install this extension, you will need:
+  1. Visual Studio version 2022 or 2019
+  2. The `Tanzu.Toolkit.VisualStudio.vsix` extension file, downloadable from any of these sources:
+      - The Visual Studio Marketplace:
+        - through the IDE by searching "Tanzu" in the "Manage Extensions" window
+        - through the website: [VS 2022](https://marketplace.visualstudio.com/items?itemName=TanzuNETExperience.TanzuToolkitForVisualStudio2022), [VS 2019](https://marketplace.visualstudio.com/items?itemName=TanzuNETExperience.TanzuToolkitForVisualStudio2019)
+      - vsixgallery.com: [VS 2022](https://www.vsixgallery.com/extension/TanzuToolkitForVisualStudio2022.ff7b6f3e-0410-4ff9-a40a-a719ee9da901), [VS 2019](https://www.vsixgallery.com/extension/TanzuToolkitForVisualStudio2019.ff7b6f3e-0410-4ff9-a40a-a719ee9da901)
```
Comment (TimHess) [inline README.md:24]: We should separate this out to a separate section or indicate that these are the "latest" builds and probably won't match the VS Marketplace or releases section

File: README.md [hunk 2]
```diff
@@ -15,10 +15,23 @@ Tanzu Toolkit for Visual Studio is an extension for Visual Studio (2022 & 2019)
...
+      - The ["Releases" section](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/releases) of this repository
+        - look for the `.vsix` file under "Assets" that corresponds to your version of Visual Studio
+
+Once you've downloaded `Tanzu.Toolkit.VisualStudio.vsix`, double-click it to prompt an installation into your instance of Visual Studio.
```
Comment (TimHess) [inline README.md:28]: This won't apply if you install through the IDE

Comment (TimHess) [reply README.md:28]:
```suggestion
If the process you selected involves saving `Tanzu.Toolkit.VisualStudio.vsix` to disk, double-click it to prompt an installation into your instance of Visual Studio.
```

Comment (atwoosnam) [reply README.md]: Oops didn't see this suggestion

Comment (atwoosnam) [reply README.md]: I rephrased the whole section in this commit: [6f26e8e](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/pull/339/commits/6f26e8ec1e9c6626811da2f18c791db4deb3c9a7) -- better?

File: README.md [hunk 3]
```diff
@@ -15,10 +15,23 @@ Tanzu Toolkit for Visual Studio is an extension for Visual Studio (2022 & 2019)
 - [Remotely debug](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/wiki/Remote-Debugging-TAS-Apps) apps running on TAS within Visual Studio
 
 ## Install
-- To install this extension, you will need Visual Studio 2022 or 2019.
-  - The extension can then be found on both versions' Visual Studio Marketplace, by searching for Tanzu Net Experience.
-- Download the `Tanzu.Toolkit.VisualStudio.vsix` file & double-click to prompt an installation into your Visual Studio instance.
-- To uninstall this extension from Visual Studio, visit the Visual Studio Extension Manager found under the `Extensions` menu.
+To install this extension, you will need Visual Studio version 2022 or 2019. This extension can be downloaded within the IDE by searching "Tanzu" in the "Manage Extensions" window
```
Comment (ccheetham) [inline README.md:18]:
```suggestion
To install this extension, you will need Visual Studio version 2022 or 2019. This extension can be downloaded within the IDE by searching "Tanzu" in the "Manage Extensions" window.
```

File: README.md [hunk 4]
```diff
@@ -15,10 +15,23 @@ Tanzu Toolkit for Visual Studio is an extension for Visual Studio (2022 & 2019)
...
+
+![Extension manager showing Tanzu Toolkit for Visual Studio 2022](https://user-images.githubusercontent.com/22666145/169391289-f8ebe6ef-8209-47ae-86e2-5ef293b19cf2.png)
+
+This extension can also be installed outside of the IDE by first downloading the appropriate `.vsix` file, then double-clicking it to prompt an installation into your instance of Visual Studio.
```
Comment (ccheetham) [inline README.md:22]:
```suggestion
This extension can also be installed outside of Visual Studio by first downloading the appropriate `.vsix` file, then double-clicking it to prompt an installation.
```

File: README.md [hunk 5]
```diff
@@ -15,10 +15,23 @@ Tanzu Toolkit for Visual Studio is an extension for Visual Studio (2022 & 2019)
...
+
+The `Tanzu.Toolkit.VisualStudio.vsix` extension file is included in each [release](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/releases) in this repository.
+
+Alternatively, latest builds can be found on vsixgallery.com, though these may contain features still under development:
```
Comment (ccheetham) [inline README.md:26]:
```suggestion
Alternatively, latest builds can be found on [vsixgallery.com](https://www.vsixgallery.com/), though these may contain features still under development:
```

File: README.md [hunk 6]
```diff
@@ -15,10 +15,23 @@ Tanzu Toolkit for Visual Studio is an extension for Visual Studio (2022 & 2019)
...
+  - [VS 2022](https://www.vsixgallery.com/extension/TanzuToolkitForVisualStudio2022.ff7b6f3e-0410-4ff9-a40a-a719ee9da901)
+  - [VS 2019](https://www.vsixgallery.com/extension/TanzuToolkitForVisualStudio2019.ff7b6f3e-0410-4ff9-a40a-a719ee9da901)
+
+## Uninstall
+To uninstall this extension from Visual Studio, visit the "Extension Manager" found under the `Extensions` menu.
```
Comment (ccheetham) [inline README.md:31]:
```suggestion
To uninstall this extension from Visual Studio, visit the "Extension Manager" found under the `Extensions` menu.
This extension can be uninstalled from Visual Studio using  "Manage Extensions" window.
```

**Review by ccheetham — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (atwoosnam) [reply README.md]: Changed in cd43144

**Review by ccheetham — APPROVED** [review]
lgtm

**Pull request merged**